tests/OptionalTest.php

```diff
@@ -296,6 +295,28 @@ public function orElseThrowTheEmptyValue()
         $result = $empty_optional->orElseThrow('\\BadFunctionCallException');
     }
 
+    /**
+     * @test
+     * @expectedException RuntimeException
+     * @expectedExceptionMessage '\ArrayObject' not implement Throwable.
+     */
+    public function orElseThrowTheNotThrowable()
+    {
+        $empty_optional = Optional::void();
```

Comment:
__[phpmd]__
* Avoid using static access to class 'PhpOptional\Optional' in method 'orElseThrowTheNotThrowable'.